### PR TITLE
New plots

### PR DESCRIPTION
**Pull request by ElenaPochernina**
Move implementation of the plots to the IDD sources. Add sample pages for each of the moved plot to the IDD samples incl. general sample page. Restore functionality of PlotLists control by making CreatePlotCardContent to return plot.GetLegend(). (bug 1087)
Reorder plots in a plotlist.

### CODE REVIEW

File: samples/Boxplot sample.html [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+    
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.min.js"></script>
+    <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/rxjs/3.1.2/rx.lite.min.js"></script>
+    <script src="../dist/idd.js"></script>
+    <script type="text/javascript">
+        $(document).ready(function () {
+            var markers = InteractiveDataDisplay.asPlot("chart").get("markers");
+
+            var data2 = $("#npz_csv").html();
```
Comment (msdvoits) [inline samples/Boxplot sample.html]: IDD plots (incl. markers) can read data from the plot host element, see https://github.com/predictionmachines/InteractiveDataDisplay/blob/master/samples/Markers.html.

For information, there is InteractiveDataDisplay.readCsv method; so you shouldn't add custom implementation  of csv parser to each sample. See https://github.com/predictionmachines/InteractiveDataDisplay/blob/master/src/idd/idd.readers.js.

Comment (msdvoits) [inline samples/Boxplot sample.html]: Elena, please remove custom csv reader implementations from your samples and use the built-in approach, as in https://github.com/predictionmachines/InteractiveDataDisplay/blob/master/samples/Markers.html.

File: src/idd/idd.markers.js [hunk 2]
```diff
@@ -26,8 +26,8 @@ InteractiveDataDisplay.Markers = function (div, master) {
 
     var _data = {};
     var _shape;
-    var _colorPalette, _sizePalette;
-    var _colorRange, _sizeRange;
+    var _colorPalette, _uncertainColorPalette, _sizePalette;
```
Comment (dvoits) [inline src/idd/idd.markers.js]: @ElenaPochernina why did you need another color and size palettes in addition to the existing? 
Is there a case when they are used together in a same plot?

File: samples/Boxplot sample.html (hunk 1 above)
Comment (dvoits) [inline samples/Boxplot sample.html]: @ElenaPochernina, please remove custom csv reader implementations from your samples and use the built-in approach, as in https://github.com/predictionmachines/InteractiveDataDisplay/blob/master/samples/Markers.html.
